fk/teamjoin.go

```diff
@@ -69,6 +73,26 @@ func teamJoin(teamUUID uuid.UUID) exitCode {
 
 }
 
+func ensureNoExistingRequests(teamUUID uuid.UUID, fingerprint fpr.Fingerprint) exitCode {
+	existingRequest, err := db.GetExistingRequestToJoinTeam(teamUUID, fingerprint)
+	if err != nil {
```

Comment:
this catches `ErrRequestNotFound` and therefore prevents you getting to line 82:

![image](https://user-images.githubusercontent.com/254290/54557346-d058d600-49b2-11e9-938e-32750917fdd9.png)

Reply:
Good spot!

Do you think having this `ErrRequestNotFound` on the query is correct?

Perhaps we should just return no error, but nil request?

Reply:
`nil` request with no error works for me

Reply:
👍

Reply:
Because of this change https://github.com/fluidkeys/fluidkeys/pull/411 this error won't raise on no errors now.